test/e2e/windows/host_process.go

```diff
@@ -154,8 +156,142 @@ var _ = SIGDescribe("[Feature:WindowsHostProcessContainers] [Excluded:WindowsDoc
 		}
 		framework.ExpectEqual(p.Status.Phase, v1.PodSucceeded)
 	})
+
+	ginkgo.It("metrics should report count of started and failed to start HostProcess containers", func() {
+		ginkgo.By("Selecting a Windows node")
+		targetNode, err := findWindowsNode(f)
+		framework.ExpectNoError(err, "Error finding Windows node")
+		framework.Logf("Using node: %v", targetNode.Name)
+
+		ginkgo.By("Getting initial kubelet metrics values")
+		beforeMetrics, err := getCurrentHostProcessMetrics(f, targetNode.Name)
+		framework.ExpectNoError(err, "Error getting initial kubelet metrics for node")
+		framework.Logf("Initial HostProcess container metrics -- StartedContainers: %v, StartedContainersErrors: %v, StartedInitContainers: %v, StartedInitContainersErrors: %v",
```

Comment:
What happens when other hostprocess tests are run in parallel?

Reply:
I the see comment below.  I guess it is generally preferred to make tests not serial?  What do other metric based tests do?

Reply:
Right we don't have a test pass that tests HostProcess containers and runs serial tests.
There were not a ton of tests that validated metrics but some were serial and others were not.